account_financial_report_qweb/__openerp__.py

```diff
@@ -35,8 +35,11 @@
         'report/templates/trial_balance.xml',
         'view/account_view.xml'
     ],
-    'test': [
-    ],
+    'external_dependencies': {
+        "python": [
+            "xlsxwriter",
+        ],
+    },
```

Comment:
And it should be split in 2 modules: one for the QWeb part, and the other for XLSX support extension.